egs2/TEMPLATE/tts1/tts.sh

```diff
@@ -201,7 +214,8 @@ tts_exp="${expdir}/tts_${tag}"
 if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
     log "Stage 1: Data preparation for data/${train_set}, data/${dev_set}, etc."
     # [Task dependent] Need to create data.sh for new corpus
-    local/data.sh ${local_data_opts}
+    local/data.sh --train_set ${train_set} --dev_set ${dev_set} --eval_set ${eval_set} \
```

Comment:
This style is not bad, maybe better than current recipe, but in asr recipe we doesn't support such arguments.
We also plan to remove phone-char option at data preparation , so for now, please change as following:

```bash
# run.sh
trans_type=phone
train_set="${trans_type}_train_nodev"
dev_set="${trans_type}_dev"
eval_sets="${trans_type}_eval"
tts.sh \
     --local_data_opts "--trans_type ${trans_type}" \
     --train_set "${train_set}" \
     ...
```

Reply:
Okay.

Reply:
Sorry, my comment may be not not clear.
Please stop to give local/data.sh the --train_set, dev_set, eval_sets for now because the other recipe doesn't support it.
It can be specified only by  `--local_data_opts "--trans_type ${trans_type}"`.

Reply:
Thanks for your comments. Please check if this commit is okay now. tx!